.github/workflows/integration.yml

```diff
@@ -16,6 +16,8 @@ on:
       - policy-test/**
       - viz/**
       - '!.devcontainer/**'
+      - rust-toolchain.toml
+      - bin/rust-toolchain-version
```

Comment:
Most of the workflows that use `bin/rust-toolchain-version` should run on the dev container. Does it make sense to write this in the `justfile` instead? AFAIK we can run just recipes as bash scripts. Might make it less awkward since we've been moving some of the script logic inside the justfiles and we've been trying to move away from maintaining the scripts 🤷🏻  I'm good either way but wdyt.

Reply:
I'm pretty sure the one where we're installing Rust is specifically because we can't run in the devcontainer...